src/components/Badge/index.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import cc from 'classcat';
+import Icon from 'components/Icon';
 
-export const VALID_TYPES = ['update', 'count'];
+export const VALID_TYPES = ['update', 'count'] as const;
```

Comment:
No requested change here, but I wonder if we'll want to start using `enum`s for things like this rather than arrays of values. There would be less extra typing for a more verbose access.

```typescript
export enum ValidTypes {
  update = 'update',
  count = 'count',
}

export interface BadgeProps {
  ...
  type?: ValidTypes;
}
```

Reply:
This is a good point. The only reason not to do this would be if there are any parts of the code that expect VALID_TYPES to exists and to be an array. I've seen code like that in storybook files.

Reply:
There are ways to loop / map over an enum, or convert an enum to an array, but yeah it's more involved (not that we should or shouldn't do it, just building context)